src/scrabBag.js

```diff
@@ -0,0 +1,21 @@
+export default function scrabBag(theInput, tileDistro = {}) {
```

Comment:
As a matter of style, consider renaming `theInput` to simply `input`.  Having said that, I would try to find an identifier that really communicated the purpose of this input - maybe `lettersPlayed`?